### PR TITLE
Fix 21905 : typo in intellisense

### LINKED ISSUE #21905: Internal Typo With JavaScript Intellisense
_From @Gisgar3 on February 12, 2018 21:39_

While working on a bot for Discord, I noticed that there was a typo in the Intellisense results for JavaScript. The below image shows the MessageBox with the typo: 
bug1  

Within the MessageBox, you can see that 'their' is misspelled as 'thier.' Simple mistake, but thought it was okay to make an issue for.

_Copied from original issue: Microsoft/vscode#43541_

### PR DESCRIPTION
**Pull request by lankaapura**
Fixes #21905

### CODE REVIEW

File: lib/lib.d.ts [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ interface ReadonlyArray<T> {
       */
     toString(): string;
     /**
-      * Returns a string representation of an array. The elements are converted to string using thier toLocalString methods.
```
Comment (mhegazy) [inline lib/lib.d.ts]: files under lib/* are not meant to be manually changed. they are generated by `jake LKG`. so please revert these changes

Comment (lankaapura) [reply lib/lib.d.ts]: @mhegazy thanks for the review. src/lib/* is the correct place right? And do I need to run `jake LKG`  and commit the changes? it generates quite a big changeset.
